assets/stylesheets/responsive/custom.scss

```diff
@@ -458,21 +458,16 @@ a.link_button_green_large {
 
 div.correspondence {
   background-color: $correspondence-bg;
-  padding-bottom: 0.5em;
-  .event_actions {
-    margin-bottom: 0;
```

Comment:
This commit description isn't particularly useful.

Why is it "better"? What have we done and why?